Resources/WiPi.plist

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>items</key>
+	<array>
+		<dict>
+			<key>cell</key>
+			<string>PSGroupCell</string>
+			<key>label</key>
+			<string>Settings</string>
+			<key>footerText</key>
+			<string>If it is enabled, WiPi will be executed by pressing and holding Wi-Fi on the Control Center or Flipswitch.</string>
```

Comment:
This doesnt flow very nicely, its kind of long. Bensge came up with "Activate WiPi by long-pressing Wi-Fi-toggle in Control Center or Flipswitch", I think its more straighforward.